src/containers/Ideas/ExplorePage.js

```diff
@@ -18,7 +19,15 @@ export const ExplorePage = (props) => {
   const [projects, setProjects] = useState([])
   const { user, deepLink, setDeepLink, signOut } = useAuthorization()
 
-  const userImage = props?.route?.params?.userImage
+  useEffect(() => {
+    // initially setting profile context
+    setProfileContext({
+      userId: user.pk,
+      userName: user.username,
+      userImage: { uri: user.user_image },
+      userImageFallback: { uri: user.user_image_fallback }
+    })
+  }, [])
 
```

Comment:
This might be a stupid question, but I am wondering: we already have the auth context above and then use the user from that context to set the profile context, right? Couldnt we also just use the auth context? Or is that because we dont want to pass all the other stuff?

Reply:
Very good question. Actually first I did it in `AuthProvider` but found that pretty packed with stuff. So didn't want to make this bigger. That is why I ended up putting it into its own context, hoping it makes maintaining it easier.

Maybe wrapping the `AuthProvider` with `ProfileProvider` instead of `App`?

Reply:
Haha, I am not sure what that means, but sounds good? :see_no_evil:

Reply:
yes reading the last question again I don't know what I meant. Classic end of day crap...

What I basically tried to say is, I think it is easier to change things later if we have it in its own context, rather than merging it with the AuthContext. Hope that makes more sense...